app/javascript/modules/listingDetails/ListingDetailsPreferences.tsx

```diff
@@ -89,7 +89,7 @@ export const ListingDetailsPreferences = ({ listingID }: ListingDetailsPreferenc
                 PREFERENCES_IDS.neighborhoodResidence,
               "Rent Burdened / Assisted Housing Preference": PREFERENCES_IDS.assistedHousing,
               "Live or Work in San Francisco Preference": PREFERENCES_IDS.liveWorkInSf,
-              "Alice Griffith Housing Development Resident": PREFERENCES_IDS.aliceGriffith,
+              "Alice Griffith Housing Development Resident": PREFERENCES_IDS.rightToReturn,
```

Comment:
suggestion (non-blocking): since we're cleaning up the use of 'Alice Griffith', we should also change the string in the anchor map from `"Alice Griffith Housing Development Resident"` to `"HOPE SF Right to Return Preference"`

Reply:
I don't think we can, because we use the preference name from salesforce as the key when we index this object. That would require more logic later when we index the object

Reply:
Right, I just realized that this component is on the Listing Details page. We should add a regression test in the review instructions, to check that the link for Alice Griffith listings is working as intended.